code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -658,7 +658,7 @@
 
 /datum/reagent/medicine/oculine/on_mob_life(mob/living/carbon/affected_mob, delta_time, times_fired)
 	affected_mob.adjust_blindness(-2 * REM * delta_time * normalise_creation_purity())
-	affected_mob.adjust_blurriness(-2 * REM * delta_time * normalise_creation_purity())
+	affected_mob.adjust_eye_blur(-4 SECONDS * REM * delta_time * normalise_creation_purity())
```

Comment:
im not sure if this is correct. the original version didnt have a timer by default afaict and this one does, but it keeps the process loop used in the old one. wouldnt this increase rate at which blurriness lowers over time?